### PR TITLE
Identity Auth integration

### PR DESCRIPTION
**Pull request by balbinas**

### CODE REVIEW

File: config/initializers/omniauth.rb [hunk 1]
```diff
@@ -1,4 +1,7 @@
 Rails.application.config.middleware.use OmniAuth::Builder do
   provider :facebook, ENV['FACEBOOK_KEY'], ENV['FACEBOOK_SECRET'],
            scope: [:email, :public_profile], secure_image_url: true, image_size: :large
+  provider :identity,
+           fields: [:name, :email],
```
Comment (balbinas) [inline config/initializers/omniauth.rb]: subir este arriba

File: app/models/identity.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class Identity < OmniAuth::Identity::Models::ActiveRecord
+  validates_presence_of :name
+  validates_uniqueness_of :email
+end
```
Comment (balbinas) [inline app/models/identity.rb]: agregar password minimum length

File: app/views/identities/login.html.erb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<%= form_tag "/auth/identity/callback" do %>
+  <div class="field">
+    <%= label_tag :auth_key, "Email" %><br>
+    <%= text_field_tag :auth_key %>
```
Comment (acrogenesis) [inline app/views/identities/login.html.erb]: cambie este field a `email_field_tag`

File: app/views/identities/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      </ul>
+    </div>
+  <% end %>
+  <div class="field">
+    <%= label_tag :name %><br>
+    <%= text_field_tag :name, @identity.try(:name) %>
+  </div>
+  <div class="field">
+    <%= label_tag :email %><br>
+    <%= text_field_tag :email, @identity.try(:email) %>
```
Comment (acrogenesis) [inline app/views/identities/new.html.erb]: aca tmb

File: app/views/identities/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+      <ul>
+      <% @identity.errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+  <div class="field">
+    <%= label_tag :name %><br>
+    <%= text_field_tag :name, @identity.try(:name) %>
```
Comment (acrogenesis) [inline app/views/identities/new.html.erb]: cambialo a `@identity&.name`

File: app/views/identities/new.html.erb (hunk 4 above)
Comment (acrogenesis) [inline app/views/identities/new.html.erb]: cambialo a `@identity&.email`

**Pull request merged**